src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -226,10 +229,16 @@ class ContentSidebar extends PureComponent<Props, State> {
      * @param {Error} error - Error object
      * @return {void}
      */
-    errorCallback = (error: $AxiosXHR<any>): void => {
+    errorCallback = (error: Error): void => {
         /* eslint-disable no-console */
         console.error(error);
         /* eslint-enable no-console */
+
+        /* eslint-disable react/prop-types */
```

Comment:
Is this to avoid checking if `onError` is defined? Seems like we'd want to leave eslint on if possible here.

Reply:
It seems like its an eslint bug. I have it in the flow types above but it still complains